src/Generator.cpp

```diff
@@ -865,7 +865,17 @@ int generate_filter_main(int argc, char **argv, std::ostream &cerr) {
     for (const auto &lib : split_string(flags_info["-p"], ",")) {
         if (lib.empty()) continue;
 #ifdef _WIN32
-        if (LoadLibrary(lib.c_str()) != nullptr) {
+        HMODULE loaded_lib;
+
+        {
+            int wide_len = MultiByteToWideChar(CP_UTF8, 0, lib.c_str(), -1, nullptr, 0);
+            auto *wide_lib = new wchar_t[wide_len];
```

Comment:
suggestion: use `std::vector<wchar_t>` here instead to get automatic cleanup; `.data()` is guaranteed to return a pointer to contiguous storage